src/v/cluster/partition_balancer_types.h

```diff
@@ -106,6 +106,7 @@ enum class partition_balancer_status {
     starting,
     ready,
     in_progress,
+    waiting_for_reports,
```

Comment:
"Waiting for reports" is overly specific and too low-level for something that we show to the user. We also should keep the number of different status values low. I think it is okay to use the "starting" status for now.

Reply:
fixed